### DISCUSSION TITLE
About clangd refactor operation

### DISCUSSION THREAD

**Discussion by Hebe970217** [clangd/clangd#1064] [Q&A]
I am here https://clangd.llvm.org/features#refactoring,
You can see that clangd supports refactoring. 
Besides rename, what other operations does refactoring support now?

**Comment by Hebe970217** [comment]
Hello?

**Reply by HighCommander4** [reply]
Please keep in mind that this is an open-source project. No one is being paid to answer every question that's posted, especially not with a 24-hour turnaround time.

You can see the refactoring-related code [here](https://searchfox.org/llvm/source/clang-tools-extra/clangd/refactor). Most of the other refactorings are in the 'tweaks' subdirectory.

**Reply by Hebe970217** [reply]
thank u
